benefit/data/sheets/CaseTypeTab/CaseTypeTab.json

```diff
@@ -139,7 +139,8 @@
   {"LiveFrom": "29/07/2018", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "TabID": "notePadTab_sscs_superuser", "TabLabel": "Notepad", "TabDisplayOrder": 15, "CaseFieldID": "appealNotePad", "TabFieldDisplayOrder": 1},
   {"LiveFrom": "29/07/2018", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-judge", "TabID": "notePadTab_sscs_judge", "TabLabel": "Notepad", "TabDisplayOrder": 15, "CaseFieldID": "appealNotePad", "TabFieldDisplayOrder": 1},
   {"LiveFrom": "29/07/2018", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-registrar", "TabID": "notePadTab_sscs_registrar", "TabLabel": "Notepad", "TabDisplayOrder": 15, "CaseFieldID": "appealNotePad", "TabFieldDisplayOrder": 1},
-  {"LiveFrom": "04/08/2018", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs", "TabID": "CaseBundles_sscs", "TabLabel": "Bundles", "TabDisplayOrder": 16, "CaseFieldID": "caseBundles", "TabFieldDisplayOrder": 1},
+  {"LiveFrom": "04/08/2018", "CaseTypeID": "Benefit", "TabID": "CaseBundles_sscs", "TabLabel": "Bundles", "TabDisplayOrder": 16, "CaseFieldID": "caseBundles", "TabFieldDisplayOrder": 1},
+  {"LiveFrom": "04/08/2018", "CaseTypeID": "Benefit", "TabID": "CaseBundles_sscs", "TabLabel": "Bundles", "TabDisplayOrder": 16, "CaseFieldID": "historicalBundles", "TabFieldDisplayOrder": 1},
```

Comment:
Should the display order be the same as caseBundles or should it be incremented?